client/cmd/dexcctl/simnet-setup.sh

```diff
@@ -2,18 +2,22 @@
 # Set up DCR and BTC wallets and register with the DEX.
 # dcrdex, dexc, and the wallet simnet harnesses should all be running before
 # calling this script.
+
+RPC_PASS="abc"
```

Comment:
Is the motivation for these changes so you can more easily edit the rpc pass?  If so, perhaps we should do `RPC_PASS="${1:-abc}"` or similar.

Reply:
I don't think the rpcserver works anymore if you don't set a password. Does it? So the pass must be supplied for the commands. But your suggestion is better and I will do that.

Reply:
Oh, I bet you were affected by the move of dexcctl.conf from ~/.dexc to ~/.dexcctl recently.

Reply:
ah, that's what changed? hmm. I guess this is not needed then, I should just make a conf file there.

Reply:
I think this is still useful enough. And I already made the pr... Feel free to close.

Reply:
Yeah this is still nice.  Will test drive it first though